tools/clang/test/CodeGenHLSL/Samples/DX11/FluidCS11_BuildGridCS.hlsl

```diff
@@ -1,4 +1,4 @@
-// RUN: %dxc -E main -T cs_6_0 %s | FileCheck %s
+﻿// RUN: %dxc -E main -T cs_6_0 %s | FileCheck %s
```

Comment:
I can't tell what's change here.

Reply:
It's the Byte Order Mark (BOM) that indicates the format of the characters and the titular byte-order (endianness). It's a non-printing character. The code already checks for this to determine what conversion method to use, if any. Because this starts as utf8, it skips a few conversions.